tests/test_attrs.py

```diff
@@ -8,7 +8,7 @@
 from zarr.attrs import Attributes
 from zarr.storage import KVStore, DirectoryStore
 from zarr._storage.v3 import KVStoreV3
-from .util import CountingDict, CountingDictV3
+from tests.util import CountingDict, CountingDictV3
```

Comment:
this is a bit of surprising change. Is it required? Is this what hatch suggests packages do? Generally, I prefer a relative imports within the tests directory.

Reply:
Oh yes, this was just a personal preference. Happy to revert it!

Reply:
Let's revert it. Happy to discuss this change elsewhere but let's keep this pr scope as small as possible.